### PR TITLE
update the description

### PR DESCRIPTION
**Pull request by khamlyuk**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,15 +3,32 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# Data Grid for Blazor - How to edit a row on a separate page
+
+This example shows how to create an edit form on a separate page.
+
+![Edit form on a separate page](images/datagrid-with-external-edit-form.gif)
+
+The example implements the following steps:
```
Comment (dirk-pieterse) [inline Readme.md]: I'm not sure how an example can implement 'steps'

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md [hunk 2]
```diff
@@ -3,15 +3,32 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# Data Grid for Blazor - How to edit a row on a separate page
+
+This example shows how to create an edit form on a separate page.
+
+![Edit form on a separate page](images/datagrid-with-external-edit-form.gif)
+
+The example shows how to implement the following:
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
The example shows how to do the following:

```

Comment (dirk-pieterse) [reply Readme.md]: However, you've already got `This example shows how to`, and now you've got the same again here `The example shows how to` - this is causing confusion

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md [hunk 3]
```diff
@@ -3,15 +3,32 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T802173)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# Data Grid for Blazor - How to edit a row on a separate page
+
+This example shows how to create an edit form on a separate page.
```
Comment (khamlyuk) [inline Readme.md]:
```suggestion
This example creates an edit form on a separate page.

```

**Pull request merged**